roles/common/tasks/networking.yml

```diff
@@ -91,22 +91,38 @@
 - name: Check if interface has its own config file
```

Comment:
can you group below os == 'ubuntu' tasks under one block with when: os == 'ubuntu'